### ISSUE TITLE
Display Event Description?

### ISSUE THREAD

**Issue by Emalton** [niccokunzmann/open-web-calendar#298]
Hello, I would like to take one of [these calendars](https://www.cms4schools.com/pro/hotlunch_ical_subscribe.cfm?detailid=836745&categoryid=all&facultyflag=menu) generated by CMS4Schools and display the description which contains the food service menu. Is there a way to display each event's description?

**Comment by niccokunzmann** [comment]
Hi, at the moment, the description is not displayed but only the summary. Only when you click on it, you can see the description. Where would you like to see it? In the Agenda view? That seems to be the only place where you have enough space.

**Comment by niccokunzmann** [comment]
What do you think about this? Could you provide a picture that shows how you would like to have it?

**Comment by niccokunzmann** [comment]
Is this a duplicate of https://github.com/niccokunzmann/open-web-calendar/issues/173?

**Comment by Emalton** [comment]
> Hi, at the moment, the description is not displayed but only the summary. Only when you click on it, you can see the description. Where would you like to see it? In the Agenda view? That seems to be the only place where you have enough space.

It would be great to have either the week view or agenda view. 

> What do you think about this? Could you provide a picture that shows how you would like to have it?

![Screenshot_20240406_135435](https://github.com/niccokunzmann/open-web-calendar/assets/9328458/e4a9f4ac-cb8d-4e7a-9f13-b00b6d7b5bda)

Link:
https://www.reachdane.org/parents/reach-dane-menus.cfm?memberid=815&month=4&year=2024&view=cal_view&categoryid=7&submit_button=View+Menu#d836745



> Is this a duplicate of #173?
Yeah this is a duplicate, sorry about that forgot I submitted it!

**Comment by niccokunzmann** [comment]
I add the ICS here because it might change: 
[food.ics](https://github.com/niccokunzmann/open-web-calendar/files/14896974/food.ics.txt)

**Comment by Emalton** [comment]
> I add the ICS here because it might change: [food.ics](https://github.com/niccokunzmann/open-web-calendar/files/14896974/food.ics.txt)

Good call, thanks! Please let me know if you need more info. Appreciate it!

**Comment by gramakri** [comment]
> Only when you click on it, you can see the description

I can't see it when I click it. Is it supposed to be visible in the overlay/popup note that appears? From the demo showing german public holidays, the ICS has a description but it doesn't appear below. There is a `DESCRIPTION:Visit https://calendarlabs.com/holidays/us/new-years-day.php to know more about New Year's Day. \n\n Like us on Facebook: http://fb.com/calendarlabs to get updates` in the ics

<img width="765" height="588" alt="Image" src="https://github.com/user-attachments/assets/79853d6c-e050-4354-8c7d-39840b09dca5" />

**Comment by andrewe24** [comment]
> > Only when you click on it, you can see the description
> 
> I can't see it when I click it. Is it supposed to be visible in the overlay/popup note that appears? From the demo showing german public holidays, the ICS has a description but it doesn't appear below. There is a `DESCRIPTION:Visit https://calendarlabs.com/holidays/us/new-years-day.php to know more about New Year's Day. \n\n Like us on Facebook: http://fb.com/calendarlabs to get updates` in the ics
> 
> <img alt="Image" width="765" height="588" src="https://private-user-images.githubusercontent.com/82041/532055867-79853d6c-e050-4354-8c7d-39840b09dca5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk1NzE4NTksIm5iZiI6MTc2OTU3MTU1OSwicGF0aCI6Ii84MjA0MS81MzIwNTU4NjctNzk4NTNkNmMtZTA1MC00MzU0LThjN2QtMzk4NDBiMDlkY2E1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTI4VDAzMzkxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4NGUzYmQzMzI1YTgxYmE1OWJjOWUyNmNjZjI3OWM0YTRkNWU3YWNkNTJjNzk1MjYyMTdkYTUxNTcxNDkyMWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HQWQUO4iroOpP9Ib6rlXUWdu32ogGMh4ps7cshXbaWw">

Same here.  My calendars have links in the description, but I can't get to them from this interface.
